docs/standard/library.md

```diff
@@ -108,21 +118,24 @@ Profile-based PCL compatibility is provided by the [Microsoft.NETCore.Portable.C
 
 Profile-based PCLs packaged as `netstandard` are easier to consume than typically packaged profile-based PCLs. `netstandard` packaging is compatible with existing users.
 
-You can see the set of PCL profiles that are compatible with the .NET Standard: 
-
-| Profile | .NET Platform Standard version |
-| ---------| --------------- |
-| Profile7  .NET Portable Subset (.NET Framework 4.5, Windows 8) | 1.1 |
-| Profile31 .NET Portable Subset (Windows 8.1, Windows Phone Silverlight 8.1)| 1.0 |
-| Profile32 .NET Portable Subset (Windows 8.1, Windows Phone 8.1) | 1.2 |
-| Profile44 .NET Portable Subset (.NET Framework 4.5.1, Windows 8.1) | 1.2 |
-| Profile49 .NET Portable Subset (.NET Framework 4.5, Windows Phone Silverlight 8) | 1.0 |
-| Profile78 .NET Portable Subset (.NET Framework 4.5, Windows 8, Windows Phone Silverlight 8) | 1.0 |
-| Profile84 .NET Portable Subset (Windows Phone 8.1, Windows Phone Silverlight 8.1) | 1.0 |
-| Profile111 .NET Portable Subset (.NET Framework 4.5, Windows 8, Windows Phone 8.1) | 1.1 |
-| Profile151 .NET Portable Subset (.NET Framework 4.5.1, Windows 8.1, Windows Phone 8.1) | 1.2 |
-| Profile157 .NET Portable Subset (Windows 8.1, Windows Phone 8.1, Windows Phone Silverlight 8.1) | 1.0 |
-| Profile259 .NET Portable Subset (.NET Framework 4.5, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8) | 1.0 |
+You can see the set of PCL profiles that are compatible with the .NET Standard:
+
+| PCL Profile | .NET Standard | PCL Platforms
+|:-----------:|:-------------:|------------------------------------------------------------------------------
+| 7           | 1.1           | .NET Framework 4.5, Windows 8
+| 31          | 1.0           | Windows 8.1, Windows Phone Silverlight 8.1
+| 32          | 1.2           | Windows 8.1, Windows Phone 8.1
+| 44          | 1.2           | .NET Framework 4.5.1, Windows 8.1
+| 49          | 1.0           | .NET Framework 4.5, Windows Phone Silverlight 8
+| 78          | 1.0           | .NET Framework 4.5, Windows 8, Windows Phone Silverlight 8
+| 84          | 1.0           | Windows Phone 8.1, Windows Phone Silverlight 8.1
+| 111         | 1.1           | .NET Framework 4.5, Windows 8, Windows Phone 8.1
+| 151         | 1.2           | .NET Framework 4.5.1, Windows 8.1, Windows Phone 8.1
+| 157         | 1.0           | Windows 8.1, Windows Phone 8.1, Windows Phone Silverlight 8.1
+| 259         | 1.0           | .NET Framework 4.5, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8
+
+Note:
+    .NET Profile Subset full name is Profile + \<PCL Profile Number from the table\>. For example: Profile7, Profile259, etc...
```

Comment:
Should I include the full Profile name like `Profile7` within the table cells or just a note like this?